remotespark/livyclientlib/livysession.py

```diff
@@ -68,16 +68,20 @@ def start(self):
 
         self.logger.debug("Session '{}' started.".format(self.kind))
 
-    def create_sql_context(self):
+    def create_sql_context(self, msg_printer):
         """Create a sqlContext object on the session. Object will be accessible via variable 'sqlContext'."""
         if self.started_sql_context:
             return
 
         self.logger.debug("Starting '{}' sql and hive session.".format(self.kind))
 
+        msg_printer.print_message('Creating SQL Context...')
```

Comment:
I do like that, I think we should open a new enhancement to track this work, it'll be nice from the user perspective to see a meaningful waiting statement.